app/src/main/java/com/aptoide/uploader/apps/StoreManager.java

```diff
@@ -33,15 +36,19 @@ public StoreManager(InstalledAppsProvider installedAppsProvider,
         .subscribeOn(scheduler);
   }
 
+  public Completable startService() {
+    return Completable.fromAction(() -> serviceBackgroundService.enable());
+  }
+
   public Completable upload(List<InstalledApp> apps) {
     return storeNameProvider.getStoreName()
         .flatMapCompletable(storeName -> languageManager.getCurrentLanguageCode()
-            .flatMapCompletable(languageCode -> Observable.fromIterable(apps)
+            .flatMapCompletable(languageCode -> startService().andThen(Observable.fromIterable(apps)
```

Comment:
maybe this startservice stopped being necessary as we started using the chain handleBackgroundService